common/domain-setter.sub.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>A page that will likely be same-origin-domain but not same-origin</title>
+
+<script>
+"use strict";
+document.domain = "{{host}}";
```

Comment:
(I mentioned that scenario in another comment, but it doesn't seem to be tested.)